helm/fullnode/values.yaml

```diff
@@ -7,15 +7,19 @@ diem_chains:
   testnet:
     id: TESTNET
     waypoint: 0:3139c30efe6dbde4228efb9df32c137dc3a2490b97ab6a086897be1d0cb336f0
-    seed_addrs:
+    seeds:
       4223dd0eeb0b0d78720a8990700955e1:
-      - "/dns4/fn.testnet.diem.com/tcp/6182/ln-noise-ik/d29d01bed8ab6c30921b327823f7e92c63f8371456fb110256e8c0e8911f4938/ln-handshake/0"
+        addresses:
+        - "/dns4/fn.testnet.diem.com/tcp/6182/ln-noise-ik/d29d01bed8ab6c30921b327823f7e92c63f8371456fb110256e8c0e8911f4938/ln-handshake/0"
+        keys:
+        - "d29d01bed8ab6c30921b327823f7e92c63f8371456fb110256e8c0e8911f4938"
+        role: "Upstream"
 
 rust_log: info
 
 image:
   repo: diem/validator
-  tag: release-1.1_a1abfc34
+  tag: devnet
```

Comment:
I'm not sure about this; I'd imagine the main use of this chart is to run a PFN for mainnet once we launch, which isn't going to be running a master rev for a long time still. If we want to support both testnet and mainnet we probably need to make this image tag chain specific.

Reply:
Agreed on making the image tag chain specific. For now though since we only have testnet, I think it makes sense to use the `devnet` tag to validate. Once mainnet launches, we can include different chain-specific image tags (or even different sample values files).

Reply:
I'll go ahead and land this if no other comments @mgorven. Tracking chain-specific image tags here: https://github.com/diem/partners/issues/860